### PR TITLE
Cache message to Debug logs

### PR DESCRIPTION
**Pull request by benaadams**
## Changes

- Move cache messages to Debug rather than Warm

#### What types of changes does your code introduce?

- [x] Bugfix (a non-breaking change that fixes an issue)

#### Requires testing

- [x] No

### CODE REVIEW

**Review by LukaszRozmej — CHANGES_REQUESTED** [review]
Identify and potentially fix the root cause

**Review by LukaszRozmej — APPROVED** [review]

File: src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs [hunk 1]
```diff
@@ -133,9 +133,10 @@ the previous head state.*/
                 }
                 else
                 {
-                    if (preWarmer?.ClearCaches() ?? false)
+                    CacheType result = preWarmer?.ClearCaches() ?? default;
+                    if (result != default)
                     {
-                        if (_logger.IsWarn) _logger.Warn("Low txs, caches are not empty. Clearing them.");
+                        if (_logger.IsDebug) _logger.Debug($"Low txs, caches {result} are not empty. Clearing them.");
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs:139]: keep it as Warn though?

**Review by LukaszRozmej — CHANGES_REQUESTED** [review]
keep as warnings for now?

File: src/Nethermind/Nethermind.Consensus/Processing/BlockCachePreWarmer.cs [hunk 2]
```diff
@@ -30,9 +30,10 @@ public Task PreWarmCaches(Block suggestedBlock, Hash256? parentStateRoot, Access
     {
         if (preBlockCaches is not null)
         {
-            if (preBlockCaches.ClearCaches())
+            CacheType result = preBlockCaches.ClearCaches();
+            if (result != default)
             {
-                if (_logger.IsWarn) _logger.Warn("Caches are not empty. Clearing them.");
+                if (_logger.IsDebug) _logger.Debug($"Caches {result} are not empty. Clearing them.");
```
Comment (benaadams) [inline src/Nethermind/Nethermind.Consensus/Processing/BlockCachePreWarmer.cs:36]:
```suggestion
                if (_logger.IsDebug) _logger.Warn($"Caches {result} are not empty. Clearing them.");
```

File: src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs (hunk 1 above)
Comment (benaadams) [inline src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs:139]:
```suggestion
                        if (_logger.IsDebug) _logger.Warn($"Low txs, caches {result} are not empty. Clearing them.");
```

Comment (benaadams) [reply src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs]: Changed to warn

**Review by LukaszRozmej — COMMENTED** [review]
But you left `IsDebug`?

**Comment by benaadams** [comment]
> But you left `IsDebug`?

Changed back to Warn

**Pull request merged**